### DISCUSSION TITLE
Must have classes that provide inherit value

### DISCUSSION THREAD

**Discussion by owuzan** [tailwindlabs/tailwindcss#4153] [Ideas]
According to the given code, it works as in the first picture, and the second picture is intended to work. Pay attention to the corners of the child element.

For example, I give an element "rounded-2xl" and I want to give the same to a child element. Whenever I make changes to the parent element, it will make our job easier to have a class that provides the value of "border-radius: inherit" instead of doing it on the child element.

Also in this case I might not want to overflow: hidden the parent element. This problem will also be resolved.

![image](https://user-images.githubusercontent.com/23698814/115735717-5ebe9380-a393-11eb-9750-cddd919e0cf9.png)

``` jsx
<div className="flex justify-center items-center h-screen w-screen bg-gradient-to-br from-orange-500 via-red-500 to-rose-500">
  <div className="bg-white rounded-2xl w-1/2 h-1/2 p-8">
    <div className="bg-gray-200 rounded-inherit w-2/3 h-2/3 flex items-center justify-center">
      <p className="font-bold text-4xl text-gray-800">Child Element</p>
    </div>
  </div>
</div>
```

![image](https://user-images.githubusercontent.com/23698814/115736238-cc6abf80-a393-11eb-8daf-78816ffbbcf2.png)

**Comment by dance2die** [comment]
I would expect Tailwind CSS to follow CSS inheritance rules.

The initial value of `border-radius` (for all four sides) is `0` and I'd expect child elements to have such value, not inherited border radii.
https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius#formal_definition

The idea looks like it can be solved by `descendant` selectors, not available in Tailwind CSS.
